torch/lib/build_all.sh

```diff
@@ -36,14 +36,20 @@ else
     LDFLAGS="$LDFLAGS -Wl,-rpath,\$ORIGIN"
 fi
 C_FLAGS="$BASIC_C_FLAGS $LDFLAGS"
+NVCC_TARGETS="THC THCS THCUNN"
 function build() {
   mkdir -p build/$1
   cd build/$1
+  BUILD_C_FLAGS=''
+  eval 'case '$1' in
+      '${NVCC_TARGETS//" "/" | "}') BUILD_C_FLAGS=$C_FLAGS;;
```

Comment:
It fixes ppc builds without CUDA. I finally got access to a CUDA-capable machine and let the tests run overnight. ppc CUDA build is still broken. I'll look into that right now and ping you later. Thanks for point this out.